src/processor/marketplace-events/index.ts

```diff
@@ -56,7 +56,10 @@ export class MarketplaceEventsProcessor extends EventProcessor {
       )
 
       this.manager = manager
-      this.gcHandler = errorHandler(collectPinsClosure(this.manager), loggingFactory('gc'))
+      this.gcHandler = composeGc([
```

Comment:
Hmm I would refactor this to have it as part of the `gc.ts` file and here only calling the main function. If you will be searching what is GCed, you won't be searching for it here... Also then you don't have to compose it twice, here and in the blockchain events processor...

Reply:
Agree. Good point